src/Resources/views/CRUD/base_list.html.twig

```diff
@@ -196,10 +198,17 @@ file that was distributed with this source code.
                                             <ul class="dropdown-menu">
                                                 {% for format in export_formats %}
                                                 <li>
-                                                    <a href="{{ admin.generateUrl('export', admin.modelmanager.paginationparameters(admin.datagrid, 0) + {'format' : format}) }}">
+                                                    {# NEXT_MAJOR: Remove completely next "a" tag and uncomment the other one #}
```

Comment:
Just to define a convention, IMO we should use "element" for HTML elements (like in `document.getElementBy*()` ES API), since Twig is already using the term "tag" in its own syntax.